rasa/cli/initial_project/config.yml

```diff
@@ -1,44 +1,13 @@
-# The config recipe.
-# https://rasa.com/docs/rasa/model-configuration/
 recipe: default.v1
-
-# The assistant project unique identifier
-# This default value must be replaced with a unique assistant name within your deployment
```

Comment:
The id should probably stay shouldn’t it as well as the comments

Reply:
doesn't the id get generated if there isn't one? seems counterintuitive to me that we have a hardcoded value in the template project?

Reply:
mhm yeah that is actually a good point - let me check who introduced that

Reply:
@ancalita any particular reason why the assistant id is hardcoded rather than generated in the initial project? https://github.com/RasaHQ/rasa/pull/12053